src/metrikker/iatjenester.ts

```diff
@@ -83,28 +97,34 @@ export const sendIaTjenesteMetrikk = async (iatjeneste: IaTjenesteMetrikk) => {
     }
 };
 
-export const useSendIaTjenesteMetrikkMottattVedSidevisningEvent = () => {
+export const useSendIaTjenesteMetrikkMottattVedSidevisningEvent = (
+    kilde: IaTjenesteKilde = IaTjenesteKilde.SYKEFRAVÆRSSTATISTIKK,
+    sendMetrikker: boolean = true
+) => {
     const context = useContext(iaTjenesterMetrikkerContext);
     const orgnr = useOrgnr();
 
     useEffect(() => {
-        const iaTjenesteMetrikk = byggIaTjenesteMottattMetrikk(orgnr);
+        const iaTjenesteMetrikk = byggIaTjenesteMottattMetrikk(orgnr, kilde);
         if (
             !erIaTjenesterMetrikkerSendtForBedrift(
                 orgnr ?? '',
-                context.bedrifterSomHarSendtMetrikker
-            )
+                context.bedrifterSomHarSendtMetrikker,
+                kilde
+            ) &&
+            sendMetrikker
         ) {
             sendIaTjenesteMetrikk(iaTjenesteMetrikk).then((isSent) => {
                 if (isSent) {
                     context.setBedrifterSomHarSendtMetrikker(
                         iaTjenesterMetrikkerErSendtForBedrift(
                             orgnr,
-                            context.bedrifterSomHarSendtMetrikker
+                            context.bedrifterSomHarSendtMetrikker,
```

Comment:
Denne linja er på SYVENDE indent..... Jeg skal unngå å gjøre noen referanser til Dantes inferno, der tallet syv spiller en vesentlig rolle. 

Uncle Bob hadde nok ikke vært like sympatisk. 🤓

Reply:
Kan nok definere funksjonen i then til en const og referere til den i stedet for å skrive den 'inline'

Reply:
Sorry for denne litt teite kommentaren @MalazAlkoj, det er jo ikke din skyld at Prettier formaterer på denne måten 🙈